lib/__tests__/needlessDisables.test.js

```diff
@@ -1,153 +1,142 @@
 'use strict';
 
 const path = require('path');
+const { stripIndent } = require('common-tags');
+
 const replaceBackslashes = require('../testUtils/replaceBackslashes');
 const standalone = require('../standalone');
-const stripIndent = require('common-tags').stripIndent;
 
 function fixture(name) {
-	return replaceBackslashes(path.join(__dirname, './fixtures/disableOptions', name));
+	return replaceBackslashes(path.join(__dirname, 'fixtures', 'disableOptions', name));
 }
 
-it('needlessDisables simple case', () => {
+it('needlessDisables simple case', async () => {
 	const config = {
 		rules: { 'block-no-empty': true },
 	};
 
 	const css = stripIndent`
-    /* stylelint-disable */
-    a {}
-    /* stylelint-enable */
-    a {
-      b {} /* stylelint-disable-line block-no-empty */
-    }
-    /* stylelint-disable */
-    a { color: pink; }
-    /* stylelint-enable */
-    a {
-      b { color: pink; } /* stylelint-disable-line block-no-empty */
-    }
-    `;
-
-	return standalone({
+		/* stylelint-disable */
+		a {}
+		/* stylelint-enable */
+		a {
+			b {} /* stylelint-disable-line block-no-empty */
+		}
+		/* stylelint-disable */
+		a { color: pink; }
+		/* stylelint-enable */
+		a {
+			b { color: pink; } /* stylelint-disable-line block-no-empty */
+		}
+		`;
+
+	const { results } = await standalone({
 		config,
 		code: css,
 		reportNeedlessDisables: true,
-	}).then((linted) => {
-		const results = linted.results;
-
-		expect(results).toHaveLength(1);
-		const warnings = results[0].warnings;
-
-		expect(warnings).toEqual([
-			{
-				line: 7,
-				column: 1,
-				text: 'Needless disable for "all"',
-				rule: '--report-needless-disables',
-				severity: 'error',
-			},
-			{
-				line: 11,
-				column: 22,
-				text: 'Needless disable for "block-no-empty"',
-				rule: '--report-needless-disables',
-				severity: 'error',
-			},
-		]);
 	});
+
+	expect(results).toHaveLength(1);
+	expect(results[0].warnings).toEqual([
+		{
+			line: 7,
+			column: 1,
+			text: 'Needless disable for "all"',
+			rule: '--report-needless-disables',
+			severity: 'error',
+		},
+		{
+			line: 11,
+			column: 21,
```

Comment:
[note] This `column` change from 22 to 21 is caused by re-indentation with TABs.